packages/blockchain-wallet-v4-frontend/src/index.html

```diff
@@ -3,27 +3,47 @@
-    <meta property="og:title" content="Blockchain.com Wallet - The World's Most Trusted Wallet">
-    <meta property="og:description" content="The world's most trusted and popular crypto wallet. Sign up today to buy and sell Bitcoin, Ethereum and other top cryptocurrencies.">
-    <meta property="og:image" content="https://www.blockchain.com/static/img/home/opengraph.png">
-    <meta property="og:image:type" content="image/png">
-    <meta name="twitter:card" content="summary_large_image">
-    <meta name="twitter:image" content="https://www.blockchain.com/static/img/home/opengraph.png">
-    <meta name="twitter:description" content="The world's most trusted and popular crypto wallet. Sign up today to buy and sell Bitcoin, Ethereum and other top cryptocurrencies.">
-    <title>Blockchain.com Wallet</title>
+    <meta name="apple-itunes-app" content="app-id=493253309" />
+    <meta
+      name="description"
+      content="The world's most trusted and popular crypto wallet. Sign up today to buy and sell Bitcoin, Ethereum and other top cryptocurrencies."
+    />
+    <meta
+      name="keywords"
+      content="bitcoin wallet, blockchain wallet, online bitcoin wallet, bitcoin wallet online"
+    />
+    <meta property="og:url" content="https://www.login.blockchain.com" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Blockchain.com Wallet - The World's Most Trusted Wallet" />
+    <meta
+      property="og:description"
+      content="The world's most trusted and popular crypto wallet. Sign up today to buy and sell Bitcoin, Ethereum and other top cryptocurrencies."
+    />
+    <meta property="og:image" content="https://www.blockchain.com/static/img/home/opengraph.png" />
+    <meta property="og:image:type" content="image/png" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:image" content="https://www.blockchain.com/static/img/home/opengraph.png" />
+    <meta
+      name="twitter:description"
+      content="The world's most trusted and popular crypto wallet. Sign up today to buy and sell Bitcoin, Ethereum and other top cryptocurrencies."
+    />
+    <title>Blockchain.com</title>
     <link rel="manifest" href="./manifest.webmanifest" />
     <script nonce="**CSP_NONCE**">
       window.APP_VERSION = '**APP_VERSION**'
       window.CAPTCHA_KEY = '**RECAPTCHA_KEY**'
       window.NONCE = '**CSP_NONCE**'
     </script>
-    <script defer nonce="**CSP_NONCE**" src="https://www.google.com/recaptcha/enterprise.js?render=**RECAPTCHA_KEY**"></script>
-    <script defer nonce="**CSP_NONCE**" src="https://www.googletagmanager.com/gtm.js?id=GTM-KK99TPJ"></script>
+    <script
+      defer
+      nonce="**CSP_NONCE**"
+      src="https://www.google.com/recaptcha/enterprise.js?render=**RECAPTCHA_KEY**"
+    ></script>
+    <script
+      defer
+      nonce="**CSP_NONCE**"
+      src="https://www.googletagmanager.com/gtm.js?id=GTM-KK99TPJ"
+    ></script>
```

Comment:
i think we should undo all changes to this file. we still want the full title here for SEO. its okay if the page title flickers imo